cylc/flow/hostuserutil.py

```diff
@@ -107,7 +107,7 @@ def get_local_ip_address(target):
     @staticmethod
     def get_host_ip_by_name(target):
         """Return internal IP address of target."""
-        return socket.gethostbyname(target)
+        return socket.getfqdn(target)
```

Comment:
I don't think this is supposed to be here @oliver-sanders ? ;)

Reply:
Actually that is the right way around, the real question is how did it get the wrong way around ;)

Reply:
haha

Reply:
😳